haas/rest.py

```diff
@@ -110,7 +110,9 @@ def rest_call(method, path, schema):
 
     * path - the url-path to map the function to. The format is the same as for
             flask's router (e.g. '/foo/<bar>/baz')
-    * method - the HTTP method for the api call (e.g. POST, GET...)
+    * method - the HTTP method for the api call (e.g. POST, GET...).
+               Can also pass a list of methods to support multiple (used in
+               testing)
```

Comment:
Ack, forgot about the naming thing. You can *almost* stack the decorators :P 

I'm fine with keeping it, since it's a step closer to what flask does anyway, and I'm a fan of not writing more of our own web framework than actually makes sense. Maybe down the line we'll put a pair of braces around everything and just pass it straight through to flask.